pkg/localmetrics/localmetrics.go

```diff
@@ -226,27 +226,27 @@ func (c *MetricsCollector) collect() {
 	}
 
 	for _, account := range accounts.Items {
-		if account.Status.Claimed {
+		if account.Spec.Claimed {
 			claimed = "true"
 		} else {
 			claimed = "false"
 		}
 
-		if account.Status.Reused {
+		if account.Spec.Reused {
 			reused = "true"
 		} else {
 			reused = "false"
 		}
 
-		if !account.Status.Claimed && account.Status.Reused &&
-			account.Status.State == "Ready" {
+		if !account.Spec.Claimed && account.Spec.Reused &&
+			account.Spec.State == "Ready" {
```

Comment:
Not your code, but can we replace this "Ready" string with the const equivalent  - if for some bizzare reason the value of 'Ready' changed in the future, we'd still be left with this hardcoded string.